### PR TITLE
Extended lorenz

### PR DESCRIPTION
**Pull request by ferrine**
following #142 

- [ ] mobius_add
- [ ] mobius_matvec
- [ ] mobius_scalar_mul
- [ ] mobius_pointwise_mul
- [ ] mobius_fn_apply

- [ ] test new functions 
- [ ] re-review Rasul's implementation

### CODE REVIEW

File: geoopt/manifolds/lorentz/__init__.py [hunk 1]
```diff
@@ -159,6 +157,7 @@ def transp_follow_expmap(
     def geodesic_unit(
         self, t: torch.Tensor, x: torch.Tensor, u: torch.Tensor, *, dim=-1, project=True
     ) -> torch.Tensor:
+        # FIXME: project U
```
Comment (ferrine) [inline geoopt/manifolds/lorentz/__init__.py:160]: @rrkarim is this intended? I missed this in my PR

Comment (rrkarim) [reply geoopt/manifolds/lorentz/__init__.py:160]: you want to add additional argument to project on the tangent space?

Comment (rrkarim) [reply geoopt/manifolds/lorentz/__init__.py:160]: You can see in the example notebook, I was projecting u before parallel transport.

Comment (ferrine) [reply geoopt/manifolds/lorentz/__init__.py:160]: I was confused about this. I think the argument should already belong to tangent space, protection should be on user side.

**Comment by ferrine** [comment]
// Paused because of the conference deadline

**Pull request closed without merging**